banzai/preview.py

```diff
@@ -0,0 +1,76 @@
+from banzai import dbs
+from banzai.utils import file_utils
+from banzai import logs
+from banzai.utils.image_utils import image_passes_criteria
+
+logger = logs.get_logger(__name__)
+
+
```

Comment:
Should `PreviewModeListener` be moved here?

Reply:
Indeed.

Reply:
Oh. Right. Circular imports make this really complicated. This will be refactored once we merge end of night and preview modes.